code/modules/jobs/job_types/geneticist.dm

```diff
@@ -19,6 +19,8 @@
 	display_order = JOB_DISPLAY_ORDER_GENETICIST
 	threat = 1.5
 
+	starting_skills = list(/datum/skill/numerical/surgery = STARTING_SKILL_SURGERY_MEDICAL)
```

Comment:
why doesn't the geneticist get the affinity when the other non-surgeon jobs also get it?

Reply:
because they're a science department employee :^)

Reply:
@Linzolle the real answer is i forgot

the answer i'd rather use is what @Poojawa said
they're like
not as Medical as the other Medical Dudes 

i mean i'll change it if you want :^)

Reply:
eh i dont really care was just wondering why lol